src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -217,7 +255,33 @@ public override async Task RunAsync()
 
             var settings = SelfHostWebHostSettingsFactory.Create(Environment.CurrentDirectory);
 
-            (var listenUri, var baseUri, var certificate) = await Setup();
+            // Determine if middleware (Easy Auth) is enabled
+            var middlewareAuthSettings = _secretsManager.GetMiddlewareAuthSettings();
+            bool authenticationEnabled = middlewareAuthSettings.ContainsKeyCaseInsensitive(Constants.MiddlewareAuthEnabledSetting) &&
+                middlewareAuthSettings[Constants.MiddlewareAuthEnabledSetting].ToLower().Equals("true");
+
+            (var listenUri, var baseUri, var certificate, string certPath, string certPassword) = await Setup();
+
+            int originalPort = Port;
+            if (authenticationEnabled)
+            {
+                // If it is enabled, then the Functions Host needs a different port
+                Port = Port + 2;
+            }
+
+            // Regardless of whether or not auth is enabled, clients should send requests here
+            var originalListenUri = listenUri.SetPort(originalPort);
+            var originalBaseUri = baseUri.SetPort(originalPort);
```

Comment:
same thing here, I think it'll be clearer if you called them `hostListenUri` and `authListenUri`